net/mdns-repeater/src/etc/inc/plugins.inc.d/mdnsrepeater.inc

```diff
@@ -28,12 +28,21 @@
 
 function mdnsrepeater_enabled()
 {
-    $mdns_repeater = new \OPNsense\MDNSRepeater\MDNSRepeater();
-    if ((string)$mdns_repeater->enabled == '1') {
-        return true;
-    }
+    $model = new \OPNsense\MDNSRepeater\MDNSRepeater();
+    return (string)$model->enabled == '1';
+}
+
+function mdnsrepeater_carp_enabled()
```

Comment:
I’d refer moving this to the only caller, also _interfaces suffix is too generic and might be used by something else already